client/src/components/BottomNavigation/BottomNavigation.js

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+// Ui components
+import { BottomNavigation, BottomNavigationAction } from "@mui/material";
+// Icons
+import FolderIcon from "@mui/icons-material/Folder";
```

Comment:
Could you group all the icons into a single import, to follow the same pattern in other components? MUI uses a custom name from the actual import location, like below, so just import using the `as` keyboard to keep the custom names MUI used

import { Folder as FolderIcon, etc } from "@mui/icons-material/";